terraform/monitoring/istio_service_setup.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+from google.cloud import monitoring_v3
+
+def getIstioServiceName(service_name, project_id, zone):
+	return 'ist:' + project_id + '-zone-' + zone + '-cloud-ops-sandbox-default-' + service_name
+
+def findService(client, service_name, project_id, zone):
+	found_service = False
+	full_service_name = getIstioServiceName(service_name, project_id, zone)
+	service = client.service_path(project_id, full_service_name)
+	while not found_service:
```

Comment:
We should put some kind of optional retry loop here, and make sure a friendly error is displayed after it fails (probably outside of this script)

It was ok to loop forever in the other test scripts, because eventually GitHub Actions will timeout/we'll catch the error. If this script is to be run as part of install.sh, we want to make sure it's more friendly

Reply:
I've added a parameter called service_wait where when set means we'll wait forever for the services. I'm a little iffy about the naming of the variable so let me know if it needs to be adjusted. I have it default set to false.